docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -0,0 +1,186 @@
+===== Option 4 - Use an IAM role Amazon Resource Name (ARN):
+An IAM role ARN is an IAM identity that you can create in your AWS account. You define the role's permissions.
+Roles do not have standard long-term credentials such as passwords or access keys.
+Instead, when you assume a role it provides you with temporary security credentials for your session.
+An IAM role's ARN can be used to specify which AWS IAM role to use to generate temporary credentials.
+For more details see the https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html[AssumeRole API documentation].
+Follow Amazon's instructions to https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html[create an IAM user], and define its permissions using the JSON permissions policy above.
+To use an IAM role's ARN, you need to provide either a <<kspm-use-a-shared-credentials-file,credential profile>> or <<kspm-use-keys-directly,access keys>> along with the `ARN role`.
+The `ARN Role` value specifies which AWS IAM role to use for generating temporary credentials.
+
+NOTE: If `ARN Role` is present, the integration will check if `Access key ID` and `Secret Access Key` are present.
+If not, the package will check for a `Credential Profile Name`.
+If a `Credential Profile Name` is not present, the default credential profile will be used.
+
+
+[[kspm-setup-eks-step-3]]
+[discrete]
+==== Finish configuring the KSPM integration for EKS
+Once you've provided AWS credentials, finish configuring the KSPM integration:
+
+1. If you want to monitor Kubernetes clusters that aren’t yet enrolled in fleet, select *New Hosts* under “where to add this integration”.
+2. Name the {agent} policy. Use a name that matches the purpose or team of the cluster(s) you want to monitor, for example, `IT-dev-k8s-clusters`.
+3. Click *Save and continue*, then *Add agent to your hosts*. The *Add agent* wizard appears and provides a DaemonSet manifest `.yaml` file with pre-populated configuration information, such as the `Fleet ID` and `Fleet URL`.
+
+[[kspm-setup-eks-step-4]]
+[discrete]
+==== Modify and deploy the DaemonSet to your clusters
+The *Add agent* wizard helps you deploy a DaemonSet on the Kubernetes clusters you wish to monitor. To do this, for each cluster:
+
+1. Download the manifest and make any necessary revisions to its configuration to suit the needs of your environment.
+2. Apply the manifest using the `kubectl apply -f` command. For example: `kubectl apply -f elastic-agent-managed-kubernetes.yaml`
+
+After a few minutes, a message confirming the {agent} enrollment appears, followed by a message confirming that data is incoming. You can then click *View assets* to see where the newly-collected configuration information appears throughout {kib}, including the <<findings-page,Findings page>> and the <<cloud-posture-dashboard, Cloud Posture dashboard>>.
+
+
+[discrete]
+[[kspm-setup-unmanaged]]
+=== Set up KSPM for unmanaged Kubernetes clusters
+
+To install the integration:
```

Comment:
It might help to provide a little more context, mostly to reiterate that the reader (probably?) doesn't need to do this if they're using EKS-managed clusters